manifests/pnp4nagios/config.pp

```diff
@@ -0,0 +1,69 @@
+class icinga2::pnp4nagios::config inherits icinga2::pnp4nagios {
+
+  $monitoring_type = 'icinga'
+  $log_type = 'syslog'
+  $debug_lvl = 1
+
+  file {$htpasswd_config:
+    path   => "$httpd_path/$htpasswd_config",
```

Comment:
why file { $htpasswd_config : }? Why not use file { "$httpd_path/$htpasswd_config": } instead?